src/actions/projects.js

```diff
@@ -39,34 +38,43 @@ const collaboratorParams = {
 }
 
 export function fetchProjects() {
-    return (dispatch) => { 
+    return (dispatch) => {
         return new Promise((resolve, reject) => {
             dispatch(addProjectsRequest)
-            getAuthUser().then( (userProfile) => {
+            getAuthUser().then((userProfile) => {
                 const userIsLoggedIn = userProfile !== null
                 let projectCalls = []
                 let allProjects = []
-                
-                // Fetch production Projects
-                projectCalls.push(apiClient.type('projects').get(productionParams).then( projects => {
-                    const taggedProjects = tagProjects(projects, false)
-                    allProjects = allProjects.concat(taggedProjects)
-                }))
+
+                let fetchPaginatedProjects = (params, _page = 1) => {
+                    return apiClient.type('projects')
+                        .get({...params, ...{page: _page}})
+                        .then((projects) => {
+                            const taggedProjects = tagProjects(projects, false)
+                            allProjects = allProjects.concat(taggedProjects)
+
+                            if (taggedProjects.length === 20) {
+                                var currentPage = _page + 1
+                                return fetchPaginatedProjects(params, currentPage)
+                            }
+                        })
+                }
+
+                projectCalls.push(fetchPaginatedProjects(productionParams));
 
                 // Fetch Beta Projects
-                projectCalls.push(apiClient.type('projects').get(betaParams).then( projects => {
+                projectCalls.push(apiClient.type('projects').get(betaParams).then(projects => {
                     const taggedProjects = tagProjects(projects, false)
                     allProjects = allProjects.concat(taggedProjects)
                 }))
```

Comment:
I think we can refactor lines 66-69 to be `projectCalls.push(fetchPaginatedProjects(betaParams))`